docker/0.2.2/fedimintd-mutinynet/docker-compose.yaml

```diff
@@ -0,0 +1,43 @@
+# Important: 127.0.0.1 should be replaced by the external ip address of the machine
+version: "3"
+
+services:
+  fedimintd:
+    image: fedimint/fedimintd:v0.2.1
```

Comment:
Should this be v0.2.2?

Reply:
nope, didn't cut that there weren't any ui changes